styles.go

```diff
@@ -2,11 +2,22 @@
 
 package promptui
 
-// Icons used for displaying prompts or status
+// These are the default icons used bu prompt-ui for select and prompts. They can either be overridden directly
+// from these variable or customized through the use of custom templates
```

Comment:
IMO we shouldn't encourage changing these globally.

Reply:
There is also a typo "by", not "bu".

In that case, should we export these? Since they're in the promptui package, they will be available anyway no?